source/Octopus.Tentacle.Tests.Integration/Util/Builders/Decorators/TentacleServiceDecoratorBuilderExtensions.cs

```diff
@@ -1,39 +1,20 @@
 using System;
+using Octopus.Tentacle.Tests.Integration.Common.Builders.Decorators;
+using Octopus.Tentacle.Tests.Integration.Common.Builders.Decorators.Proxies;
 using Octopus.Tentacle.Tests.Integration.Support;
-using Octopus.Tentacle.Tests.Integration.Util.Builders.Decorators.Proxies;
 
 namespace Octopus.Tentacle.Tests.Integration.Util.Builders.Decorators
 {
     public static class TentacleServiceDecoratorBuilderExtensions
     {
-        public static TentacleServiceDecoratorBuilder RecordMethodUsages<TService>(this TentacleServiceDecoratorBuilder builder, out IRecordedMethodUsages recordedUsages)
-            where TService : class
-        {
-            var localMethodUsages = new MethodUsages();
-            recordedUsages = localMethodUsages;
-
-            return builder.RegisterProxyDecorator<TService>(service => MethodUsageProxyDecorator.Create(service, localMethodUsages));
-        }
```

Comment:
These methods were moved into the shared library, but the other ones use the `TentacleConfigurationTestCase` which was not moved, hence why this file still remains

Reply:
Is it worth moving `TentacleConfigurationTestCase` to common?

Reply:
It's not applicable to the Kubernetes integration tests as it has information such as version, communication type, which runtime etc.